doc/conf.py

```diff
@@ -304,4 +304,5 @@ def setup(app):
     'examples_dirs': examples_dirs,
     'gallery_dirs': gallery_dirs,
     'find_mayavi_figures': find_mayavi_figures,
+    'examples_expected_to_fail': set(['../examples/plot_raise.py'])
```

Comment:
It is not really need here, because I convert this entry at the end of the build into a set. There it has to be a set, to use set operations of intersection and difference. I thought to give the hint of this being a set, as I put in the defaults value an empty set.